parser.y

```diff
@@ -231,7 +231,12 @@ rule
 
   token_declaration: id int_opt alias { result = val }
 
-  rule_declaration: "%rule" IDENTIFIER "(" rule_args ")" ":" rule_rhs_list
+  rule_declaration: "%rule" id_colon ":" rule_rhs_list
```

Comment:
I think `%rule` **without** `%inline` is needless because it is same with 
a normal production rule.
I agree to introducing zero parameters for `%inline` but it might be better to support the feature in https://github.com/ruby/lrama/pull/369.

Reply:
Indeed it is.  So I close this PR. Thank you for your review!